src/main/java/me/totalfreedom/totalfreedommod/util/History.java

```diff
@@ -1,108 +1,108 @@
-                    try
-                    {
-                        //UUIDs or playernames actually work with this one
-                        //TODO: fix the stupid api on how it's not working name histories
-                        //URL url = new URL("https://api.ashcon.app/mojang/v2/user/" + compactUuid);
-                        URL url = new URL("https://api.mojang.com/user/profiles/" + compactUuid + "/names");
-                        HttpURLConnection conn = (HttpURLConnection)url.openConnection();
-                        //conn.setRequestProperty("User-Agent", "");
-                        BufferedReader reader = new BufferedReader(new InputStreamReader(conn.getInputStream()));
-                        FName[] oldNames = gson.fromJson(reader, FName[].class);
-                        if (oldNames == null)
-                        {
-                            FSync.playerMsg(sender, ChatColor.RED + "Player not found!");
-                            return;
-                        }
-                        reader.close();
-                        conn.disconnect();
-                        Arrays.sort(oldNames);
-                        printHistory(sender, oldNames);
-                    }
-                    catch (Exception ex)
-                    {
-                        FSync.playerMsg(sender, ChatColor.RED + "Error, see logs for more details.");
-                        FLog.severe(ex);
-                    }
-                }
-                else
-                {
-                    FSync.playerMsg(sender, ChatColor.RED + "Player not found!");
-                }
-            }
-        }.runTaskAsynchronously(TotalFreedomMod.plugin());
-    }
-
-    private static void printHistory(CommandSender sender, FName[] oldNames)
-    {
-        if (oldNames.length == 1)
-        {
-            FSync.playerMsg(sender, ChatColor.GREEN + oldNames[0].getName() + ChatColor.GOLD + " has never changed their name.");
-            return;
-        }
-        FSync.playerMsg(sender, ChatColor.GOLD + "Original name: " + ChatColor.GREEN + oldNames[0].getName());
-        for (int i = 1; i < oldNames.length; i++)
-        {
-            Date date = new Date(oldNames[i].getChangedToAt());
-            String formattedDate = dateFormat.format(date);
-            FSync.playerMsg(sender, ChatColor.BLUE + formattedDate + ChatColor.GOLD + " changed to " + ChatColor.GREEN + oldNames[i].getName());
-        }
-    }
-
-    private static class FName implements Comparable<FName>
-    {
-
-        private String name;
-        private long changedToAt;
-
-        @Override
-        public int compareTo(FName other)
-        {
-            return Long.compare(this.changedToAt, other.changedToAt);
-        }
-
-        public String getName()
-        {
-            return name;
-        }
-
-        public long getChangedToAt()
-        {
-            return changedToAt;
-        }
-    }
+package me.totalfreedom.totalfreedommod.util;
```

Comment:
This entire file hasn't changed, and doing this is gonna break the Git history (OR make it difficult to work with)